tests/test_optimizers_vs_ase.py

```diff
@@ -1,126 +1,278 @@
 import copy
-import functools
+from typing import TYPE_CHECKING, Any
 
+import numpy as np
 import pytest
 import torch
-from ase.filters import FrechetCellFilter
+from ase.build import bulk
+from ase.filters import FrechetCellFilter, UnitCellFilter
 from ase.optimize import FIRE
 
 import torch_sim as ts
-from torch_sim.io import state_to_atoms
+from torch_sim.io import atoms_to_state, state_to_atoms
 from torch_sim.models.mace import MaceModel
-from torch_sim.optimizers import frechet_cell_fire
+from torch_sim.optimizers import frechet_cell_fire, unit_cell_fire
 
 
-try:
+if TYPE_CHECKING:
     from mace.calculators import MACECalculator
-except ImportError:
-    MACECalculator = None
 
 
-@pytest.mark.skipif(MACECalculator is None, reason="MACECalculator not installed")
-def test_torchsim_frechet_cell_fire_vs_ase_mace(
-    rattled_sio2_sim_state: ts.state.SimState,
+@pytest.fixture
+def osn2_sim_state(torchsim_mace_mpa: MaceModel) -> ts.state.SimState:
```

Comment:
we could add these to the top level conftest and then maybe also include in the simstate lists used to check model consistency?

I have a slight preference to all the simstate fixtures being all together

Reply:
good call, i was planning to do that but forgot